pkgs/applications/networking/feedreaders/feeds/default.nix

```diff
@@ -0,0 +1,108 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+
+, appstream
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+
+, glib
+, gtk3
+, libhandy
+, webkitgtk
+, python3
+}:
+let
+  listparser = python3.pkgs.buildPythonPackage rec {
+    pname = "listparser";
+    version = "0.18";
+
+    src = python3.pkgs.fetchPypi {
+      inherit pname version;
+      sha256 = "0hdqs1mmayw1r8yla43hgb4d9y3zqs5483vgf8j9ygczkd2wrq2b";
+    };
+
+    propagatedBuildInputs = with python3.pkgs; [
+      requests
+      six
+    ];
+
+    doCheck = false;
+
+    meta = with lib; {
+      description = "A parser for subscription lists";
+      homepage = "https://github.com/kurtmckee/listparser";
+      license = licenses.lgpl3;
+      maintainers = [
+        maintainers.pbogdan
+      ];
+      platforms = platforms.linux;
+    };
+  };
+in
+python3.pkgs.buildPythonApplication rec {
+  pname = "feeds-unstable";
```

Comment:
Why is this unstable?

Reply:
That is the convention when obtaining a git commit instead of tag, see https://nixos.org/nixpkgs/manual/#sec-package-naming. It is also controversial: https://github.com/NixOS/nixpkgs/pull/88023#issuecomment-629849960